src/raylib/model/Animation.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Animation
+*/
+
+#ifndef RAYLIB_MODEL_ANIMATION_HPP_
+#define RAYLIB_MODEL_ANIMATION_HPP_
+
+extern "C"
+{
+#include <raylib.h>
+}
+
+#include <string>
+#include "Model.hpp"
+
+namespace raylib
+{
+    namespace model
+    {
+        /// The Animation class
+        class Animation {
+          public:
+            /// Animation constructor. Used to initialize an animation from a file
+            /// File format accepted: obj and probably some other
+            ///
+            /// @param animPath the relative path to the animation to import
+            Animation(const std::string_view animPath);
```

Comment:
Thinking of it you must use std::filesystem::filepath for all... filepath.
Otherwise we will have issues since the directory separators aren't the same on unix/windows systems.